### PR TITLE
Signs Font Consistency + Extra

### PR DESCRIPTION
**Pull request by YesterdaysPromise**
## About The Pull Request

Someone asked me on discord to make a couple signs, in the same style as the current ones. Looked through them, found them bit inconsistant, fixed that, mainly the font. Made six-latter signs a thing, also. And also while I was at it, added the thing I got cooking for wallening which is waysigns with stair icons, showcasing on what floor things are. plenty useful for some multi-z maps.

image

## Why It's Good For The Game
Consistent font doesn't bother the mind as much, additional signs were requested of me, and the cross z-level waysigns are just useful for mappers and players alike.

## Changelog

:cl:
add: Added couple new signs to the game (med, medbay, morgue, chapel, viro, gene, botany)
add: Way-signs now have additional states, showing vertical directions.
image: Made the font on the signs consistant.
/:cl:

### CODE REVIEW

File: code/game/objects/structures/signs/signs_departments.dm [hunk 1]
```diff
@@ -5,18 +5,32 @@
 
 ///////MEDBAY
 
-/obj/structure/sign/departments/medbay
+/obj/structure/sign/departments/med
```
Comment (Mothblocks) [inline code/game/objects/structures/signs/signs_departments.dm:8]: having "med", "med_alt", and "medbay" seems extremely confusing

Comment (YesterdaysPromise) [reply code/game/objects/structures/signs/signs_departments.dm:8]: what do you sugget?

Comment (Mothblocks) [reply code/game/objects/structures/signs/signs_departments.dm:8]: Ugh, I see. "Med" just says med, and "Medbay" says medbay. Probably nothing better and just looks dumb.

**Comment by lessthnthree** [comment]
The equal amount of positive/negative space on those new arrows make it difficult at quick glance to determine what direction the arrow is pointing.

**Comment by YesterdaysPromise** [comment]
> The equal amount of positive/negative space on those new arrows make it difficult at quick glance to determine what direction the arrow is pointing.

those aren't arrows. They are stairs. Negative space facing the name? it's down, it looks as if you were on level with stairs coming from above. Positive space facing the name? Walk up those positive-space stairs to get to the destination.

**Comment by CoiledLamb** [comment]
needs pictures friend

**Comment by YesterdaysPromise** [comment]
> needs pictures friend

A) I feel threatened beign called a 'friend'.
B) Here you go.

File: code/game/objects/structures/signs/signs_departments.dm (hunk 1 above)
Comment (Dorsisdwarf) [reply code/game/objects/structures/signs/signs_departments.dm:8]: maybe medbay_med and medbay_med_alt?

**Review by Wallemations — APPROVED** [review]

**Comment by lessthnthree** [comment]
> > The equal amount of positive/negative space on those new arrows make it difficult at quick glance to determine what direction the arrow is pointing.
> 
> those aren't arrows. They are stairs. Negative space facing the name? it's down, it looks as if it were on level with stairs coming from above. Positive space facing the name? Walk up those positive-space stairs to get to the destination.

okay cool, I was just going by IDB thought the arrows were being replaced, not a set being added

**Pull request merged**